src/utils/url.ts

```diff
@@ -39,6 +40,16 @@ export const openRoomUrl = (url: string) => {
 
 export const enterVenue = (venueId: string) => openUrl(venueInsideUrl(venueId));
 
+export const joinVenue = (venueId?: string, hasVenueEntrance?: boolean) => {
+  if (!venueId) return;
+
```

Comment:
Should we make `venueId` a required prop here, and push the 'if no venueId do nothing' logic back to the calling side? As-is, it feels like it could be a bit confusing.

Also, potentially use named props for the 2nd param/'options', so that instead of looking like `joinVenue('foo', true)` on the calling side, it can be more semantic/self documenting: `joinVenue('foo', { hasVenueEntrace: true })`

```suggestion
export interface JoinVenueOptions {
  hasVenueEntrance?: boolean
}

export const joinVenue = (venueId: string, { hasVenueEntrance }: JoinVenueOptions) => {
```

Reply:
Sounds like a reasonable change, I like the idea.